code/datums/components/crafting/recipes.dm

```diff
@@ -44,6 +44,7 @@
 	return TRUE
 
 /datum/crafting_recipe/proc/on_craft_completion(mob/user, atom/result)
+	user.throw_alert_text(/atom/movable/screen/alert/text/smallhappy, "[result.name] has been crafted successfully.", override = FALSE) // MOJAVE SUN EDIT - FO text alert
```

Comment:
throw something in there so the name gets capitalized
![image](https://github.com/Mojave-Sun/mojave-sun-13/assets/63623569/b42dff5c-6fc3-4357-9a91-00ceb9b3ddce)

Reply:
yeah like a capital 9mm pistol...

Reply:
just slap a "The" at the start of it

Reply:
**The melt down gold bar has been crafted successfully.**